docs/src/usage.txt

```diff
@@ -9,9 +9,9 @@ Arguments:
 Options:
   -l, --list                 List all commands in the cache
   -f, --render <FILE>        Render a specific markdown file
-  -p, --platform <PLATFORM>  Override the operating system, can be specified multiple times in order
-                             of preference [possible values: linux, macos, sunos, windows, android,
-                             freebsd, netbsd, openbsd]
+  -p, --platform <PLATFORM>  Override the operating system, can be specified multiple times in order of preference
+                             [possible values: linux, macos, sunos, windows, android, freebsd, netbsd, openbsd,
+                             common]
```

Comment:
CI fails here because it executes `tldr --help` on a terminal with a different width. This is kind of annoying, and I don't know if one can easily emulate this programmatically. To fix CI, you can just manually do the adjustments that are suggested in the CI run

Reply:
Worked!